web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -55,6 +55,34 @@ export class DataStoreService {
       );
   }
 
+  /**
+   * Returns an Observable that loads and emits the projects to which the user
+   * is part of.
+   *
+   */
+  loadAccessibleProject$(userEmail: string): Observable<List<Project>> {
+    return this.db
+      .collection('projects')
+      .snapshotChanges()
+      .pipe(
+        map(actions =>
```

Comment:
Does each value emitted represent a project? If so this variable should be named as such.

Reply:
renamed to projects.